include/utils.hpp

```diff
@@ -10,6 +10,7 @@ namespace Utils
 	{
 		T x;
 		T y;
+		int cost;
```

Comment:
I don't think this parameter is required.

Reply:
Sorry, had forgotten to stash. :stuck_out_tongue: